js/script.js

```diff
@@ -60,44 +60,80 @@ class ToDoList{
             this.inputText.value = "";
             this.msgEl.innerText="";
             this.inputText.style.cssText="border-bottom: 1p solid black";
-            this.listEl.innerHTML = this.listToHtml(toDoList);
+            this.listToHtml(toDoList);
         }
         else this.notify(validToDo.msg, this.inputText);
     }
-    edit(){
+    createTodo(toDo){
+        const li = document.createElement("li");
+        const navDiv = document.createElement("div");
+        const spanEdit = document.createElement("span");
+        const spanDelete = document.createElement("span");
+        const input = document.createElement("input");
+        const doneDiv = document.createElement("div");
+
+        navDiv.setAttribute("class", "edit-delete");
+
+        spanEdit.setAttribute("class", "edit-icon");
+        spanEdit.innerHTML = `<i class="fa fa-edit"></i>`;
+
+        spanDelete.setAttribute("class", "delete-icon");
+        spanDelete.innerHTML = `<i class="fa fa-trash-o"></i>`;
+
+
+        navDiv.append(spanEdit);
+        navDiv.append(spanDelete);
+
+        li.append(navDiv);
+
+        input.setAttribute("class", "to-do-text");
+        input.setAttribute("value", toDo.name);
+        input.setAttribute("disabled", true);
+
+       
+        li.append(input);
 
+        doneDiv.setAttribute("class", "done");
+        doneDiv.innerHTML = ` <i class="fa fa-check-circle"></i>`;
+
+        li.append(doneDiv);
+
+        spanEdit.addEventListener("click", ()=>this.delete(toDo.id));
+        spanDelete.addEventListener("click", ()=>this.delete(toDo.id));
+        
+        this.listEl.append(li);
     }
-    delete(){
+    edit(){
 
+    }
+    delete(id){
+        let toDoList = this.getData();
```

Comment:
why do I need 2 identical strings 110 and 113?

Reply:
to retrive data from localStorage again

Reply:
to retrieve data again from localstorage

Reply:
i don't see changes